website/docs/r/cm_network.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "rightscale"
+page_title: "Rightscale: network"
+sidebar_current: "docs-rightscale-resource-network"
+description: |-
+  Create and maintain a rightscale cloud management network.
+---
+
+# rightscale_network
+
+Use this resource to create, update or destroy rightscale [networks](http://docs.rightscale.com/cm/dashboard/manage/networks/index.html) in cloud management.
```

Comment:
Should this link to the API docs instead? or some more generic network documentation if there is such a thing. Seems a bit strange to link to docs that explain how the UI works.